docker/coturn/turnserver.conf

```diff
@@ -157,6 +157,11 @@ tls-listening-port=5349
 min-port=49152
 max-port=65535
 
+# Socket buffer size (in bytes).
+# (default value is 2097152)
+#
+#sock-buf-size=2097152
```

Comment:
Probably would help readers to see the human friendly version of the size. This is 2mb right?